include/seqan3/alignment/pairwise/detail/pairwise_alignment_algorithm_banded.hpp

```diff
@@ -249,38 +255,6 @@ class pairwise_alignment_algorithm_banded :
      * phase the first cell of the column is computed and in the iteration phase all remaining cells are computed.
      * In the final phase the last cell is possibly evaluated for a new alignment optimum.
      * Note that the length of `sequence2` determines the size of the column.
-     *
```

Comment:
Why is this doc part deleted ? 🤔

Reply:
Not sure. I applied an older stashed commit and this might have removed this. I revert this. Thanks for spotting.